.circleci/config.yml

```diff
@@ -1,5 +1,8 @@
 version: 2.1
 
+orbs:
+  codecov: codecov/codecov@3.1.1
```

Comment:
Did you use 3.1.1 for a particular reason? The docs show 3.2.2 in the example.

Reply:
was from a copy/paste from the examples, will change it to use the latest version. But I want also to test if we can fix this by just running pytest from the project root instead of the readthedocs folder (I mean, it should also fix the problem with codecov, but don't know if there was a reason to run the tests inside that directory)